server/src/controllers/christmas-tree.es6

```diff
@@ -13,13 +14,15 @@ const jwt = require('jsonwebtoken');
 const email = require('../email/email-util.es6');
 
 const christmasTree = {};
+const Op = Sequelize.Op;
```

Comment:
What does Op stand for? Operation? do we want to name this DbOperation or something? As is it is not clear what Op refers to.

Reply:
just a sequelize thing - we can change to operator - task on 285